hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -1238,123 +1260,113 @@ public void testListStatus() throws IOException {
-          "b1", "k1", ReplicationType.RATIS,
-          ReplicationFactor.THREE);
+    OmKeyInfo omKeyInfo = TestOMRequestUtils.createOmKeyInfo("v1",
+        "b1", "k1", ReplicationType.RATIS,
+        ReplicationFactor.THREE);
 
-      // Add block to key.
-      List<OmKeyLocationInfo> omKeyLocationInfoList = new ArrayList<>();
-      Pipeline pipeline = getRandomPipeline();
+    // Add block to key.
+    List<OmKeyLocationInfo> omKeyLocationInfoList = new ArrayList<>();
+    Pipeline pipeline = getRandomPipeline();
 
-      OmKeyLocationInfo omKeyLocationInfo =
-          new OmKeyLocationInfo.Builder().setBlockID(
-              new BlockID(100L, 1000L))
-              .setOffset(0).setLength(100L).setPipeline(pipeline).build();
+    OmKeyLocationInfo omKeyLocationInfo =
+        new OmKeyLocationInfo.Builder().setBlockID(
+            new BlockID(100L, 1000L))
+            .setOffset(0).setLength(100L).setPipeline(pipeline).build();
 
-      omKeyLocationInfoList.add(omKeyLocationInfo);
+    omKeyLocationInfoList.add(omKeyLocationInfo);
 
-      OmKeyLocationInfo omKeyLocationInfo2 =
-          new OmKeyLocationInfo.Builder().setBlockID(
-              new BlockID(200L, 1000L))
-              .setOffset(0).setLength(100L).setPipeline(pipeline).build();
-      omKeyLocationInfoList.add(omKeyLocationInfo2);
+    OmKeyLocationInfo omKeyLocationInfo2 =
+        new OmKeyLocationInfo.Builder().setBlockID(
+            new BlockID(200L, 1000L))
+            .setOffset(0).setLength(100L).setPipeline(pipeline).build();
+    omKeyLocationInfoList.add(omKeyLocationInfo2);
 
-      OmKeyLocationInfo omKeyLocationInfo3 =
-          new OmKeyLocationInfo.Builder().setBlockID(
-              new BlockID(100L, 2000L))
-              .setOffset(0).setLength(100L).setPipeline(pipeline).build();
-      omKeyLocationInfoList.add(omKeyLocationInfo3);
+    OmKeyLocationInfo omKeyLocationInfo3 =
+        new OmKeyLocationInfo.Builder().setBlockID(
+            new BlockID(100L, 2000L))
+            .setOffset(0).setLength(100L).setPipeline(pipeline).build();
+    omKeyLocationInfoList.add(omKeyLocationInfo3);
 
-      omKeyInfo.appendNewBlocks(omKeyLocationInfoList, false);
+    omKeyInfo.appendNewBlocks(omKeyLocationInfoList, false);
 
-      KeyManagerImpl keyManagerImpl =
-          new KeyManagerImpl(ozoneManager, scmClientMock, conf, "om1");
+    KeyManagerImpl keyManagerImpl =
+        new KeyManagerImpl(ozoneManager, scmClientMock, conf, "om1");
 
-      keyManagerImpl.refresh(omKeyInfo);
+    keyManagerImpl.refresh(omKeyInfo);
+
+    verify(sclProtocolMock, times(1))
+        .getContainerWithPipelineBatch(containerIDs);
 
-      verify(sclProtocolMock, times(1))
-          .getContainerWithPipelineBatch(containerIDs);
-    } finally {
-      cluster.shutdown();
-    }
   }
 
 
   @Test
   public void testRefreshPipelineException() throws Exception {
-    MiniOzoneCluster cluster = MiniOzoneCluster.newBuilder(conf).build();
```

Comment:
This function has only really been changed to remove the cluster, (which has been replaced by the TestOmManager's OzoneManager.)  Removing the try block made it look like there were many other changes but they are just whitespace.